sound/soc/sof/ipc4-loader.c

```diff
@@ -112,16 +112,12 @@ static ssize_t sof_ipc4_fw_parse_ext_man(struct snd_sof_dev *sdev,
 				return -EINVAL;
 			}
 
-			/* a module's config is always the same size */
-			fw_module->bss_size = fm_config[fm_entry->cfg_offset].is_bytes;
 
 			dev_dbg(sdev->dev,
 				"module %s: UUID %pUL cfg_count: %u, bss_size: %#x\n",
 				fm_entry->name, &fm_entry->uuid, fm_entry->cfg_count,
-				fw_module->bss_size);
```

Comment:
it should be used to calculate module memory usage and much like CPC case that we have two sources of bss_size, one in module extended manifest and one in topology. Currently we only set a unknown number 280 to bss_size in topology for copier only and not set it for other modules. This function follows ref driver and should use 
fw_module->bss_size, not base_cfg->is_page or we should check base_cfg->is_page like CPC case as
````c
       if (base_cfg->is_page)
                 bss = base_cfg->is_page;
      else
                bss = fw_module->bss_size;
````

````c
static void
sof_ipc4_update_pipeline_mem_usage(struct snd_sof_dev *sdev, struct snd_sof_widget *swidget,
				   struct sof_ipc4_base_module_cfg *base_config)
{
	struct sof_ipc4_fw_module *fw_module = swidget->module_info;
	struct snd_sof_widget *pipe_widget;
	struct sof_ipc4_pipeline *pipeline;
	int task_mem, queue_mem;
	int ibs, bss, total;

	ibs = base_config->ibs;
	bss = base_config->is_pages; // should use fw_module->bss_size
````

Reply:
@RanderWang, OK, I drop this patch from the series.
Handling of this 'is_pages' is not a subject of this series and I'm anyways uncertain of what the variables:
`fm_config[fm_entry->cfg_offset].is_bytes` is in bytes, so
`fw_module->bss_size` is alsom in bytes
`base_config->is_pages` is in ????
`bss` likely should be in bytes or in ???? ? Probably bytes.
thus
`base_config->is_pages` should be in bytes?

Reply:
````c
struct sof_ipc4_base_module_cfg {
	uint32_t cpc; /* the max count of Cycles Per Chunk processing */
	uint32_t ibs; /* input Buffer Size (in bytes)  */
	uint32_t obs; /* output Buffer Size (in bytes) */
	uint32_t is_pages; /* number of physical pages used */
	struct sof_ipc4_audio_format audio_fmt;
} __packed __aligned(4);
````

Reply:
@RanderWang, thinking about this: we can still drop bss_size as we save the pointer to fw_mod_cfg and it is available over there.

Reply:
yes, agree